app/Enemizer.php

```diff
@@ -247,6 +247,8 @@ public function writeOptionsFile()
             ]);
         }
 
+        Log::debug($options);
```

Comment:
probably can remove this debug?

Reply:
Yea I'll remove that, it was left behind on accident.  Good catch.